Cloudflare worker support

Support `ulidx` operating in Cloudflare worker contexts, most likely using the monotonic factory feature as active timestamps aren't available. See #5 for more detail.

This also does not work in Vercel edge functions. The error message being:
```
✘ [ERROR] Could not resolve "node:crypto"

    node_modules/ulidx/dist/esm/index.js:1:19:
      1 │ import crypto from 'node:crypto';
        ╵                    ~~~~~~~~~~~~~

  The package "node:crypto" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this error.
```

This happens in a SvelteKit application

> This also does not work in Vercel edge functions.

@hamza1311 FWIW I am using `ulidx v2.0.0` successfully under Vercel Edge functions (thus running on Cloudflare workers). 

(With the key note that the timestamp does not increase within the same request due to Cloudflare workers' Date.now() restriction - which doesn't matter for my use-cases.)

> ✘ [ERROR] Could not resolve "node:crypto"
>    node_modules/ulidx/dist/esm/index.js:1:19:

Your error suggests you appear be using a version prior to v2 - it references `node_modules/ulidx/dist/esm/index.js`, whereas v2.0.0 compiles to `ulidx/dist/node/..` and `ulidx/dist/browser/...`.  

Vercel Edge/CF workers requires using the 'browser' import. This is selected automatically for me with ulidx v2 (using webpack+turbopack config provided by Next.JS) according to the exports configured in ulidx's `package.json`.  
(This was implemented in https://github.com/perry-mitchell/ulidx/issues/20.)

--- a/package.json
+++ b/package.json
@@ -22,6 +22,7 @@
     "./dist/node/index.cjs": "./dist/browser/index.cjs",
     "./dist/node/index.js": "./dist/browser/index.js"
   },
+  "react-native": "./dist/browser/index.cjs",
   "types": "dist/index.d.ts",
   "scripts": {
     "bench": "npm run build && node test/benchmark.js",

--- a/package.json
+++ b/package.json
@@ -76,26 +76,26 @@
   "homepage": "https://github.com/perry-mitchell/ulidx#readme",
   "devDependencies": {
     "@rollup/plugin-alias": "^5.0.0",
-    "@rollup/plugin-commonjs": "^25.0.0",
+    "@rollup/plugin-commonjs": "^25.0.3",
     "@rollup/plugin-node-resolve": "^15.1.0",
-    "@rollup/plugin-typescript": "^11.1.1",
-    "@types/node": "^20.2.5",
+    "@rollup/plugin-typescript": "^11.1.2",
+    "@types/node": "^20.4.4",
     "benchmark": "^2.1.4",
-    "c8": "^7.14.0",
+    "c8": "^8.0.0",
     "chai": "^4.3.7",
     "husky": "^4.3.8",
-    "lint-staged": "^13.2.2",
+    "lint-staged": "^13.2.3",
     "mocha": "^10.2.0",
     "mochify": "^9.2.0",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.8.8",
     "rimraf": "^5.0.1",
-    "rollup": "^3.23.1",
-    "sinon": "^15.1.0",
-    "tslib": "^2.5.3",
-    "typescript": "^5.1.3"
+    "rollup": "^3.26.3",
+    "sinon": "^15.2.0",
+    "tslib": "^2.6.0",
+    "typescript": "^5.1.6"
   },
   "dependencies": {
-    "layerr": "^2.0.0"
+    "layerr": "^2.0.1"
   }
 }

--- a/package.json
+++ b/package.json
@@ -13,6 +13,10 @@
       "browser": {
         "import": "./dist/browser/index.js",
         "require": "./dist/browser/index.cjs"
+      },
+      "worker": {
+        "import": "./dist/browser/index.js",
+        "require": "./dist/browser/index.cjs"
       }
     }
   },

